### ISSUE TITLE
nzbget 23 not supported

### ISSUE THREAD

**Issue by gielie** [clinton-hall/nzbToMedia#1984]
**Describe the bug**
Postprocessing on nzbget 23 doesnt work in combination with sickchill

**Technical Specs**
1. Running on:           synology NAS in a docker container
2. Python version      '3.11'
3. Download Client:  NZBget
4. Intended Media Management SickChill
/
**Log**
```
<html>
<body>
<!--StartFragment-->
INFO | Tue Feb 20 2024 11:31:58 | Collection Hightown.S03E04.Jackpot.1080p.AMZN.WEB-DL.DDP5.1.H.264-NTb added to history
-- | -- | --
ERROR | Tue Feb 20 2024 11:31:58 | Post-process-script nzbToSiCKRAGE for Hightown.S03E04.Jackpot.1080p.AMZN.WEB-DL.DDP5.1.H.264-NTb failed (terminated with unknown status)
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: json.decoder.JSONDecodeError: Expecting value: line 2 column 1 (char 1)
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: raise JSONDecodeError("Expecting value", s, err.value) from None
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: File "/usr/lib/python3.11/json/decoder.py", line 355, in raw_decode
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: obj, end = self.raw_decode(s, idx=_w(s, 0).end())
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: File "/usr/lib/python3.11/json/decoder.py", line 337, in decode
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: ^^^^^^^^^^^^^^^^^^^^^^^^^^
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: return _default_decoder.decode(s)
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: File "/usr/lib/python3.11/json/__init__.py", line 346, in loads
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: return complexjson.loads(self.text, **kwargs)
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: File "/nzbtomedia/libs/common/requests/models.py", line 900, in json
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: ^^^^^^^^
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: if r.json()['result'] == 'success':
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: File "/nzbtomedia/core/auto_process/tv.py", line 417, in process
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: result = tv.process(section_name, input_directory, input_name, status, client_agent, download_id, input_category, failure_link)
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: File "/nzbtomedia/core/processor/nzb.py", line 127, in process
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: ^^^^^^^^^^^^
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: return nzb.process(
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: File "/nzbtomedia/core/processor/nzbget.py", line 100, in process
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: ^^^^^^^^^^^^^^^^
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: result = nzbget.process()
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: File "/nzbtomedia/nzbToMedia.py", line 760, in main
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: result = nzbToMedia.main(sys.argv, section)
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: File "/nzbtomedia/nzbToSiCKRAGE.py", line 281, in <module>
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: Traceback (most recent call last):
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: -- Cleanup finished --
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: Returning to directory: /
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: REASON : Error: [Errno 20] Not a directory: 'git'
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: If errors occur, manual cleanup may be required.
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: WARNING: Automatic cleanup could not be executed.
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: -- Cleaning folders: ['libs', 'core'] --
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: REASON : Error: [Errno 20] Not a directory: 'git'
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: If errors occur, manual cleanup may be required.
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: WARNING: Automatic cleanup could not be executed.
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: -- Cleaning bytecode --
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: Changing to directory: /nzbtomedia
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: [10:31:57] [POSTPROCESS]::SICKRAGE: SUCCESS: The download succeeded, sending a post-process request
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: [10:31:57] [INFO]::MAIN: FLATTEN: Flattening directory: /downloads/tv/Hightown.S03E04.Jackpot.1080p.AMZN.WEB-DL.DDP5.1.H.264-NTb
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: [10:31:57] [INFO]::MAIN: SiCKRAGE:tv Could not create a fork object for default. Probaly class not added yet.
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: [10:31:57] [INFO]::MAIN: SiCKRAGE:tv fork set to default
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: [10:31:57] [INFO]::MAIN: Attempting to verify tv fork
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: [10:31:57] [INFO]::MAIN: Calling SiCKRAGE:tv to post-process:Hightown.S03E04.Jackpot.1080p.AMZN.WEB-DL.DDP5.1.H.264-NTb
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: [10:31:57] [INFO]::MAIN: Auto-detected SECTION:SiCKRAGE
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: [10:31:57] [INFO]::MAIN: Script triggered from NZBGet Version 23.0.
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: [10:31:57] [INFO]::MAIN: #########################################################
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: [10:31:57] [INFO]::MAIN: ## ..::[nzbToMedia.py]::.. ##
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: [10:31:57] [INFO]::MAIN: #########################################################
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: [10:31:57] [WARNING]::MAIN: Install ffmpeg with x264 support to enable this feature ...
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: [10:31:57] [WARNING]::MAIN: Failed to locate ffprobe. Video corruption detection disabled!
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: [10:31:57] [WARNING]::MAIN: Install ffmpeg with x264 support to enable this feature ...
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: [10:31:57] [WARNING]::MAIN: Failed to locate ffmpeg. Transcoding disabled!
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: [10:31:57] [WARNING]::MAIN: Failed to locate par2. Repair and rename using par files will not be possible!
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: [10:31:57] [INFO]::MAIN: nzbToMedia Version:12.1.12 Branch:master (Linux 4.4.302+)
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: [10:31:57] [INFO]::MAIN: No update needed
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: [10:31:57] [ERROR]::MAIN: Unable to determine installed version via git, please check your logs!
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: [10:31:57] [INFO]::MAIN: Checking if git needs an update
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: [10:31:57] [INFO]::MAIN: Checking database structure...
INFO | Tue Feb 20 2024 11:31:57 | nzbToSiCKRAGE: [10:31:57] [INFO]::MAIN: Python v3.11 will reach end of life in 1319 days.
INFO | Tue Feb 20 2024 11:31:56 | nzbToSiCKRAGE: warnings.warn("beets.mediafile is deprecated; use mediafile instead")
INFO | Tue Feb 20 2024 11:31:56 | nzbToSiCKRAGE: /nzbtomedia/libs/common/beets/mediafile.py:19: UserWarning: beets.mediafile is deprecated; use mediafile instead
INFO | Tue Feb 20 2024 11:31:55 | Executing post-process-script nzbToSiCKRAGE for Hightown.S03E04.Jackpot.1080p.AMZN.WEB-DL.DDP5.1.H.264-NTb
INFO | Tue Feb 20 2024 11:31:54 | Queueing Hightown.S03E04.Jackpot.1080p.AMZN.WEB-DL.DDP5.1.H.264-NTb for post-processing
INFO | Tue Feb 20 2024 11:31:54 | Hightown.S03E04.Jackpot.1080p.AMZN.WEB-DL.DDP5.1.H.264-NTb returned from history back to download queue
INFO | Tue Feb 20 2024 11:31:54 | Post-processing again Hightown.S03E04.Jackpot.1080p.AMZN.WEB-DL.DDP5.1.H.264-NTb

<!--EndFragment-->
</body>
</html>```

**Comment by clinton-hall** [comment]
This looks to be a permissions issue. The user that runs nzbget doesn't have access (write access) to the nzbToMedia directory and therefore can't do the git checks.

**Comment by rEes9P** [comment]
I too am running into issues with NZBGet v23 on MacOS. `which python3` gives me `Python 3.12.2` on Terminal but SickChill seems to be using Python 3.11.8 (for some reason I seem to have both installed on my system though I installed Python via Homebrew.

Here is the log:

```
Tue Feb 27 2024 18:30:08	INFO	Executing post-process-script nzbToSickBeard for ShowName.S01E01.1080p
Tue Feb 27 2024 18:30:08	INFO	nzbToSickBeard: Traceback (most recent call last):
Tue Feb 27 2024 18:30:08	INFO	nzbToSickBeard:   File "/Users/JohnDoe/Library/Application Support/NZBGet/scripts/nzbToMedia/nzbToSickBeard.py", line 275, in <module>
Tue Feb 27 2024 18:30:08	INFO	nzbToSickBeard:     import nzbToMedia
Tue Feb 27 2024 18:30:08	INFO	nzbToSickBeard:   File "/Users/JohnDoe/Library/Application Support/NZBGet/scripts/nzbToMedia/nzbToMedia.py", line 731, in <module>
Tue Feb 27 2024 18:30:08	INFO	nzbToSickBeard:     eol.check()
Tue Feb 27 2024 18:30:08	INFO	nzbToSickBeard:   File "/Users/JohnDoe/Library/Application Support/NZBGet/scripts/nzbToMedia/eol.py", line 102, in check
Tue Feb 27 2024 18:30:08	INFO	nzbToSickBeard:     warn_for_status(version, grace_period)
Tue Feb 27 2024 18:30:08	INFO	nzbToSickBeard:   File "/Users/JohnDoe/Library/Application Support/NZBGet/scripts/nzbToMedia/eol.py", line 143, in warn_for_status
Tue Feb 27 2024 18:30:08	INFO	nzbToSickBeard:     days_left = lifetime(version)
Tue Feb 27 2024 18:30:08	INFO	nzbToSickBeard:                 ^^^^^^^^^^^^^^^^^
Tue Feb 27 2024 18:30:08	INFO	nzbToSickBeard:   File "/Users/JohnDoe/Library/Application Support/NZBGet/scripts/nzbToMedia/eol.py", line 73, in lifetime
Tue Feb 27 2024 18:30:08	INFO	nzbToSickBeard:     time_left = PYTHON_EOL[(major, minor)] - now
Tue Feb 27 2024 18:30:08	INFO	nzbToSickBeard:                 ~~~~~~~~~~^^^^^^^^^^^^^^^^
Tue Feb 27 2024 18:30:08	INFO	nzbToSickBeard: KeyError: (3, 12)
Tue Feb 27 2024 18:30:08	ERROR	Post-process-script nzbToSickBeard for ShowName.S01E01.1080p failed (terminated with unknown status)
Tue Feb 27 2024 18:30:08	INFO	Collection ShowName.S01E01.1080p added to history
```

**Comment by clinton-hall** [comment]
@rEes9P This is a different issue (which should be resolved in nightly branch.)
Can you please switch to nightly branch and verify.

**Comment by rEes9P** [comment]
@clinton-hall Thanks, switching to the nightly branch resolved it! Should I switch back to master in the future?

On an aside, I remember being on the OSX branch a few years ago but don't see it now 🤔

**Comment by clinton-hall** [comment]
OSX branch was merged in and fully supported some time ago.
I'll aim to merge nightly into master soon (a few weeks at most). I just want to be sure this doesn't break too much (it DOES remove Python 2.7 and 3.7 support so may impact some NAS users. they will need to switch to the new (archived) py27-37 branch.

**Comment by rEes9P** [comment]
Super! Your scripts are rock solid. Thank you so much for your work and for supporting them over the years.

**Comment by rEes9P** [comment]
I was looking at the logs and, though the nightly build works just fine, noticed a lot of `SyntaxWarning: invalid escape sequence` messages that weren't there earlier.

```
Thu Feb 29 2024 12:31:49	INFO	Executing post-process-script nzbToSickBeard for ShowName.S01E01.480p
Thu Feb 29 2024 12:31:49	INFO	nzbToSickBeard: /Users/john_doe/Library/Application Support/NZBGet/scripts/nzbToMedia/libs/common/beets/mediafile.py:19: UserWarning: beets.mediafile is deprecated; use mediafile instead
Thu Feb 29 2024 12:31:49	INFO	nzbToSickBeard:   warnings.warn("beets.mediafile is deprecated; use mediafile instead")
Thu Feb 29 2024 12:31:49	INFO	nzbToSickBeard: /Users/john_doe/Library/Application Support/NZBGet/scripts/nzbToMedia/libs/common/transmissionrpc/client.py:92: SyntaxWarning: invalid escape sequence '\d'
Thu Feb 29 2024 12:31:49	INFO	nzbToSickBeard:   match = re.match('^(\d+):(\d+)$', item)
Thu Feb 29 2024 12:31:49	INFO	nzbToSickBeard: /Users/john_doe/Library/Application Support/NZBGet/scripts/nzbToMedia/libs/common/transmissionrpc/client.py:312: SyntaxWarning: invalid escape sequence '\d'
Thu Feb 29 2024 12:31:49	INFO	nzbToSickBeard:   version_parser = re.compile('(\d).(\d+) \((\d+)\)')
Thu Feb 29 2024 12:31:49	INFO	nzbToSickBeard: /Users/john_doe/Library/Application Support/NZBGet/scripts/nzbToMedia/libs/common/subliminal/core.py:526: SyntaxWarning: invalid escape sequence '\*'
Thu Feb 29 2024 12:31:49	INFO	nzbToSickBeard:   """Refine a video using :ref:`refiners`.
Thu Feb 29 2024 12:31:49	INFO	nzbToSickBeard: /Users/john_doe/Library/Application Support/NZBGet/scripts/nzbToMedia/libs/common/subliminal/core.py:555: SyntaxWarning: invalid escape sequence '\*'
Thu Feb 29 2024 12:31:49	INFO	nzbToSickBeard:   """List subtitles.
Thu Feb 29 2024 12:31:49	INFO	nzbToSickBeard: /Users/john_doe/Library/Application Support/NZBGet/scripts/nzbToMedia/libs/common/subliminal/core.py:596: SyntaxWarning: invalid escape sequence '\*'
Thu Feb 29 2024 12:31:49	INFO	nzbToSickBeard:   """Download :attr:`~subliminal.subtitle.Subtitle.content` of `subtitles`.
Thu Feb 29 2024 12:31:49	INFO	nzbToSickBeard: /Users/john_doe/Library/Application Support/NZBGet/scripts/nzbToMedia/libs/common/subliminal/core.py:613: SyntaxWarning: invalid escape sequence '\*'
Thu Feb 29 2024 12:31:49	INFO	nzbToSickBeard:   """List and download the best matching subtitles.
Thu Feb 29 2024 12:31:49	INFO	nzbToSickBeard: /Users/john_doe/Library/Application Support/NZBGet/scripts/nzbToMedia/libs/common/subliminal/extensions.py:8: SyntaxWarning: invalid escape sequence '\*'
Thu Feb 29 2024 12:31:49	INFO	nzbToSickBeard:   """:class:~stevedore.extensions.ExtensionManager` with support for registration.
Thu Feb 29 2024 12:31:49	INFO	nzbToSickBeard: /Users/john_doe/Library/Application Support/NZBGet/scripts/nzbToMedia/libs/common/subliminal/video.py:124: SyntaxWarning: invalid escape sequence '\*'
Thu Feb 29 2024 12:31:49	INFO	nzbToSickBeard:   """Episode :class:`Video`.
Thu Feb 29 2024 12:31:49	INFO	nzbToSickBeard: /Users/john_doe/Library/Application Support/NZBGet/scripts/nzbToMedia/libs/common/subliminal/video.py:214: SyntaxWarning: invalid escape sequence '\*'
Thu Feb 29 2024 12:31:49	INFO	nzbToSickBeard:   """Movie :class:`Video`.
Thu Feb 29 2024 12:31:49	INFO	nzbToSickBeard: /Users/john_doe/Library/Application Support/NZBGet/scripts/nzbToMedia/libs/common/subliminal/refiners/__init__.py:1: SyntaxWarning: invalid escape sequence '\*'
Thu Feb 29 2024 12:31:49	INFO	nzbToSickBeard:   """
Thu Feb 29 2024 12:31:49	INFO	nzbToSickBeard: [12:31:49] [INFO]::MAIN: Python v3.12 will reach end of life in 1676 days.
Thu Feb 29 2024 12:31:49	INFO	nzbToSickBeard: [12:31:49] [INFO]::MAIN: Checking database structure...
Thu Feb 29 2024 12:31:49	INFO	nzbToSickBeard: [12:31:49] [INFO]::MAIN: Checking if git needs an update
Thu Feb 29 2024 12:31:50	INFO	nzbToSickBeard: [12:31:50] [INFO]::MAIN: No update needed
Thu Feb 29 2024 12:31:50	INFO	nzbToSickBeard: [12:31:50] [INFO]::MAIN: nzbToMedia Version:b7d6ad8c0752832150299dbd4e38c68f59125eef Branch:nightly (Darwin 23.3.0)
Thu Feb 29 2024 12:31:50	INFO	nzbToSickBeard: [12:31:50] [WARNING]::MAIN: Failed to locate 7zip. Transcoding of disk images and extraction of .7z files will not be possible!
Thu Feb 29 2024 12:31:50	INFO	nzbToSickBeard: [12:31:50] [WARNING]::MAIN: Failed to locate par2. Repair and rename using par files will not be possible!
Thu Feb 29 2024 12:31:50	INFO	nzbToSickBeard: [12:31:50] [WARNING]::MAIN: Failed to locate ffmpeg. Transcoding disabled!
Thu Feb 29 2024 12:31:50	INFO	nzbToSickBeard: [12:31:50] [WARNING]::MAIN: Install ffmpeg with x264 support to enable this feature  ...
Thu Feb 29 2024 12:31:50	INFO	nzbToSickBeard: [12:31:50] [WARNING]::MAIN: Failed to locate ffprobe. Video corruption detection disabled!
Thu Feb 29 2024 12:31:50	INFO	nzbToSickBeard: [12:31:50] [WARNING]::MAIN: Install ffmpeg with x264 support to enable this feature  ...
Thu Feb 29 2024 12:31:50	INFO	nzbToSickBeard: [12:31:50] [INFO]::MAIN: #########################################################
Thu Feb 29 2024 12:31:50	INFO	nzbToSickBeard: [12:31:50] [INFO]::MAIN: ## ..::[nzbToMedia.py]::.. ##
Thu Feb 29 2024 12:31:50	INFO	nzbToSickBeard: [12:31:50] [INFO]::MAIN: #########################################################
Thu Feb 29 2024 12:31:50	INFO	nzbToSickBeard: [12:31:50] [INFO]::MAIN: Script triggered from NZBGet Version 23.0.
Thu Feb 29 2024 12:31:50	INFO	nzbToSickBeard: [12:31:50] [INFO]::MAIN: Auto-detected SECTION:SickBeard
Thu Feb 29 2024 12:31:50	INFO	nzbToSickBeard: [12:31:50] [INFO]::MAIN: Calling SickBeard:SickChill to post-process:ShowName.S01E01.480p
Thu Feb 29 2024 12:31:50	INFO	nzbToSickBeard: [12:31:50] [INFO]::MAIN: Attempting to auto-detect SickChill fork
Thu Feb 29 2024 12:31:50	INFO	nzbToSickBeard: [12:31:50] [INFO]::MAIN: SickBeard:SickChill fork auto-detection found custom params {'delete_on': None, 'proc_dir': None, 'force_next': None, 'force': None, 'is_priority': None, 'process_method': None}
Thu Feb 29 2024 12:31:50	INFO	nzbToSickBeard: [12:31:50] [INFO]::MAIN: SickBeard:SickChill fork set to custom
Thu Feb 29 2024 12:31:50	INFO	nzbToSickBeard: [12:31:50] [INFO]::MAIN: SickBeard:SickChill Could not create a fork object for custom. Probaly class not added yet.
Thu Feb 29 2024 12:31:50	INFO	nzbToSickBeard: [12:31:50] [INFO]::MAIN: FLATTEN: Flattening directory: /Users/john_doe/Downloads/complete/ShowName.S01E01.480p
Thu Feb 29 2024 12:31:50	INFO	nzbToSickBeard: [12:31:50] [POSTPROCESS]::SICKBEARD: SUCCESS: The download succeeded, sending a post-process request
Thu Feb 29 2024 12:31:54	INFO	nzbToSickBeard: [12:31:54] [POSTPROCESS]::SICKBEARD: Processing in folder /Users/john_doe/Downloads/complete/ShowName.S01E01.480p
Thu Feb 29 2024 12:31:54	INFO	nzbToSickBeard: [12:31:54] [POSTPROCESS]::SICKBEARD: Processing /Users/john_doe/Downloads/complete/ShowName.S01E01.480p
Thu Feb 29 2024 12:31:54	INFO	nzbToSickBeard: [12:31:54] [POSTPROCESS]::SICKBEARD: Processing folder /Users/john_doe/Downloads/complete/ShowName.S01E01.480p
Thu Feb 29 2024 12:31:54	INFO	nzbToSickBeard: [12:31:54] [POSTPROCESS]::SICKBEARD: Processing /Users/john_doe/Downloads/complete/ShowName.S01E01.480p/ShowName.S01E01.480p.mkv (ShowName.S01E01.480p)
Thu Feb 29 2024 12:31:54	INFO	nzbToSickBeard: [12:31:54] [POSTPROCESS]::SICKBEARD: Retrieving episode object for S03E06
Thu Feb 29 2024 12:31:54	INFO	nzbToSickBeard: [12:31:54] [POSTPROCESS]::SICKBEARD: Snatch history had a quality in it, using that: SD DVD
Thu Feb 29 2024 12:31:54	INFO	nzbToSickBeard: [12:31:54] [POSTPROCESS]::SICKBEARD: This was manually downloaded but it appears to be better quality than what we have so I'm marking it as priority
Thu Feb 29 2024 12:31:54	INFO	nzbToSickBeard: [12:31:54] [POSTPROCESS]::SICKBEARD: Is ep a priority download: True
Thu Feb 29 2024 12:31:54	INFO	nzbToSickBeard: [12:31:54] [POSTPROCESS]::SICKBEARD: There is no existing file so there's no worries about replacing it
Thu Feb 29 2024 12:31:54	INFO	nzbToSickBeard: [12:31:54] [POSTPROCESS]::SICKBEARD: This download is marked a priority download so I'm going to replace an existing file if I find one
Thu Feb 29 2024 12:31:54	INFO	nzbToSickBeard: [12:31:54] [POSTPROCESS]::SICKBEARD: Found release name ShowName.S01E01.480p
Thu Feb 29 2024 12:31:54	INFO	nzbToSickBeard: [12:31:54] [POSTPROCESS]::SICKBEARD: Destination folder for this episode: /Users/john_doe/Downloads/SickChill/ShowName/Season 1
Thu Feb 29 2024 12:31:54	INFO	nzbToSickBeard: [12:31:54] [POSTPROCESS]::SICKBEARD: Found the following associated files for /Users/john_doe/Downloads/complete/ShowName.S01E01.480p/ShowName.S01E01.480p.mkv: ['/Users/john_doe/Downloads/complete/ShowName.S01E01.480p/ShowName.S01E01.480p.idx', '/Users/john_doe/Downloads/complete/ShowName.S01E01.480p/ShowName.S01E01.480p.sub', '/Users/john_doe/Downloads/complete/ShowName.S01E01.480p/ShowName.S01E01.480p.nzb']
Thu Feb 29 2024 12:31:54	INFO	nzbToSickBeard: [12:31:54] [POSTPROCESS]::SICKBEARD: Deleting non-allowed associated files for /Users/john_doe/Downloads/complete/ShowName.S01E01.480p/ShowName.S01E01.480p.mkv: ['/Users/john_doe/Downloads/complete/ShowName.S01E01.480p/ShowName.S01E01.480p.nzb']
Thu Feb 29 2024 12:31:54	INFO	nzbToSickBeard: [12:31:54] [POSTPROCESS]::SICKBEARD: Deleting file /Users/john_doe/Downloads/complete/ShowName.S01E01.480p/ShowName.S01E01.480p.nzb
Thu Feb 29 2024 12:31:54	INFO	nzbToSickBeard: [12:31:54] [POSTPROCESS]::SICKBEARD: Allowing associated files for /Users/john_doe/Downloads/complete/ShowName.S01E01.480p/ShowName.S01E01.480p.mkv: ['/Users/john_doe/Downloads/complete/ShowName.S01E01.480p/ShowName.S01E01.480p.idx', '/Users/john_doe/Downloads/complete/ShowName.S01E01.480p/ShowName.S01E01.480p.sub']
Thu Feb 29 2024 12:31:54	INFO	nzbToSickBeard: [12:31:54] [POSTPROCESS]::SICKBEARD: Moving file from /Users/john_doe/Downloads/complete/ShowName.S01E01.480p/ShowName.S01E01.480p.mkv to /Users/john_doe/Downloads/SickChill/ShowName/Season 1/1x01 - Episode 1.mkv
Thu Feb 29 2024 12:31:54	INFO	nzbToSickBeard: [12:31:54] [POSTPROCESS]::SICKBEARD: Moving file from /Users/john_doe/Downloads/complete/ShowName.S01E01.480p/ShowName.S01E01.480p.idx to /Users/john_doe/Downloads/SickChill/ShowName/Season 1/1x01 - Episode 1.idx
Thu Feb 29 2024 12:31:54	INFO	nzbToSickBeard: [12:31:54] [POSTPROCESS]::SICKBEARD: Moving file from /Users/john_doe/Downloads/complete/ShowName.S01E01.480p/ShowName.S01E01.480p.sub to /Users/john_doe/Downloads/SickChill/ShowName/Season 1/1x01 - Episode 1.sub
Thu Feb 29 2024 12:31:54	INFO	nzbToSickBeard: [12:31:54] [POSTPROCESS]::SICKBEARD: Processing succeeded for /Users/john_doe/Downloads/complete/ShowName.S01E01.480p/ShowName.S01E01.480p.mkv
Thu Feb 29 2024 12:31:54	INFO	nzbToSickBeard: [12:31:54] [POSTPROCESS]::SICKBEARD: Successfully processed
Thu Feb 29 2024 12:31:54	INFO	nzbToSickBeard: [12:31:54] [INFO]::CLEANDIRS: Directory /Users/john_doe/Downloads/complete/ShowName.S01E01.480p still contains 1 unprocessed file(s), skipping ...
Thu Feb 29 2024 12:31:54	INFO	nzbToSickBeard: [12:31:54] [INFO]::MAIN: The /Users/john_doe/Library/Application Support/NZBGet/scripts/nzbToMedia/nzbToSickBeard.py script completed successfully.
Thu Feb 29 2024 12:31:54	INFO	nzbToSickBeard: Changing to directory: /Users/john_doe/Library/Application Support/NZBGet/scripts/nzbToMedia
Thu Feb 29 2024 12:31:54	INFO	nzbToSickBeard: -- Cleaning bytecode --
Thu Feb 29 2024 12:31:54	INFO	nzbToSickBeard: b'Removing __pycache__/\n'
Thu Feb 29 2024 12:31:54	INFO	nzbToSickBeard: b'Removing __pycache__/\n'
Thu Feb 29 2024 12:31:54	INFO	nzbToSickBeard: -- Cleaning folders: ['libs', 'core'] --
Thu Feb 29 2024 12:31:54	INFO	nzbToSickBeard: No folders to clean
Thu Feb 29 2024 12:31:54	INFO	nzbToSickBeard: Returning to directory:  /
Thu Feb 29 2024 12:31:54	INFO	nzbToSickBeard: -- Cleanup finished --
Thu Feb 29 2024 12:31:54	INFO	nzbToSickBeard: SickBeard: Successfully post-processed 1x01 - Episode 1.sub!
Thu Feb 29 2024 12:31:54	INFO	Post-process-script nzbToSickBeard for ShowName.S01E01.480p successful
Thu Feb 29 2024 12:31:54	INFO	Collection ShowName.S01E01.480p added to history
```

**Comment by clinton-hall** [comment]
Thanks for this.

beets.mediafile in theory is only a warning, but just imports mediafile anyway. So shouldn't have any impact, and I have now put a fix in to test if this works calling mediafile directly.

Transmission RPC only impacts Transmission users, and relates to the need to now escape the '\' or use "raw" text. I have opted to escape for now to test.

Subliminal warnings all come about due to the previously escaped '*' in the docstrings. These shouldn't impact the modules themselves, and I have now removed the escape characters.

So, if you want to test this on 'remove-warnings' branch, feel free. Otherwise I'll try and set up some tests over the weekend and then push to nightly when I am sure these are fixed.
